docs/api-docs/storefront/graphql/graphql-api-overview.md

```diff
@@ -392,6 +385,16 @@ In general, to reduce complexity, reduce the number of objects requested:
 * reduce the number of items in nested collections
 * request less fields
 
+<a id="see-it-in-action" class="devdocsAnchor"></a>
+
+## See it in Action
+
+To see the GraphQL storefront API in action, checkout the [Bootstrap + Vanilla JS Storefront API Example](https://bigcommerce.github.io/storefront-api-examples/html-bootstrap-vanillajs/) hosted on GitHub. This example shows how a static HTML site can be used to render dynamic product information via the GraphQL Storefront API.

```

Comment:
Inconsistency in the capitalization of "storefront/Storefront." The convention in this doc has been to leave the word uncapitalized. I recommend sticking with this convention for now for consistency.